demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -475,6 +483,48 @@ export class DemoMeetingApp
       }
   }
 
+  async createReplacementImageBlob(startColor: string, endColor: string): Promise<Blob> {
```

Comment:
Probably out of scope but isn't it cooler to have an option to upload a pic that will be used as the replacement image? :p

Reply:
I think the more useful next step would be to move many of these options to a popup like most UC apps do.

Reply:
We could add that in the future, but it does add a bit of complexity on the serverless app in hosting and image to be displayed.